test/index.test.js

```diff
@@ -1,3 +1,30 @@
-test('Initial test', () => {
-  expect(1).toBe(1);
+// /* eslint-disable jest/expect-expect */
+
+const request = require('supertest');
+const connection = require('../server/database/config/connection');
+const dbBuild = require('../server/database/config/build');
+
+const app = require('../server/app');
+
+beforeAll(() => {
+  return dbBuild();
+});
+
+afterAll(() => {
+  return connection.end();
+});
+
+describe('Cohort', () => {
+  test('Route /cohorts status 200, json header, data.id = data.id', (done) => {
+    return request(app)
+      .get('/api/v1/cohorts')
+      .expect(200)
+      .expect('Content-Type', /json/)
+      .end((err, res) => {
+        if (err) return done(err);
+        const { data } = res.body;
+        expect(data.id).toBe(data.id);
```

Comment:
I think there is a problem here, you expect that ( data.id to be data.id ) ! two are the same? 
also, data here is an array of objects, not object so you can't access data using dot notation
```suggestion
        expect(data).toHaveLength(0);
```